src/ol/source/wms.jsdoc

```diff
@@ -1,3 +0,0 @@
-/**
- * @namespace ol.source.wms
```

Comment:
I didn't. But since the is nothing in the wms namespace any longer, it should. I cannot try right now... already offline.